src/xGadgetron/cGadgetron/include/sirf/Gadgetron/gadgetron_data_containers.h

```diff
@@ -429,7 +475,12 @@ namespace sirf {
 	*/
 	class AcquisitionsVector : public MRAcquisitionData {
 	public:
-		AcquisitionsVector(AcquisitionsInfo info = AcquisitionsInfo())
+        AcquisitionsVector(const char* filename_with_ext)
+        {
+            this->read(std::string(filename_with_ext));
+        }
+
+        AcquisitionsVector(AcquisitionsInfo info = AcquisitionsInfo())
```

Comment:
```suggestion
        AcquisitionsVector(const AcquisitionsInfo& info = AcquisitionsInfo())
```
presumably we want to avoid copying the object